Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Adapted from: https://github.com/elasticdog/tiddlywiki-docker/blob/master/5/Dockerfile
+
+ARG NODE_LTS_VERSION=18
+
+FROM node:${NODE_LTS_VERSION}-alpine
+
+ARG TIDDLYWIKI_VERSION=5.2.7
+ARG TIDDLYWIKI_DATA_DIR=/tiddlywiki
+ARG TIDDLYWIKI_PORT=8080
+
+# https://github.com/nodejs/docker-node/blob/master/docs/BestPractices.md#handling-kernel-signals
```

Comment:
Thanks for the link, most of it is over my head, but the warning about running as root did jump out. Is that something we could address?

Reply:
I can address that! In general if you are using a minimal image (like [most `alpine` images](https://www.alpinelinux.org/about/)) there's not a lot a root user can do in the container if they gained access. Nevertheless, I agree because of "principle of least privilege" and all that. 😄